apps/dashboard/src/components/activity/activity-empty-state.tsx

```diff
@@ -0,0 +1,174 @@
+import { Button } from '@/components/primitives/button';
+import { cn } from '@/utils/ui';
+import { PlayCircleIcon } from 'lucide-react';
+import { RiCloseCircleLine } from 'react-icons/ri';
+import { motion, AnimatePresence } from 'motion/react';
+import { ExternalLink } from '../shared/external-link';
+import { useNavigate } from 'react-router-dom';
+import { buildRoute, ROUTES } from '@/utils/routes';
+import { useEnvironment } from '@/context/environment/hooks';
+
+interface ActivityEmptyStateProps {
+  className?: string;
+  emptySearchResults?: boolean;
+  onClearFilters?: () => void;
+}
+
+export function ActivityEmptyState({ className, emptySearchResults, onClearFilters }: ActivityEmptyStateProps) {
+  const navigate = useNavigate();
+  const { currentEnvironment } = useEnvironment();
+
+  const handleNavigateToWorkflows = () => {
+    navigate(buildRoute(ROUTES.WORKFLOWS, { environmentSlug: currentEnvironment?.slug ?? '' }));
+  };
+
+  return (
+    <AnimatePresence mode="wait">
+      <motion.div
+        key="empty-state"
+        className={cn('flex h-full w-full items-center justify-center', className)}
+        initial={{ opacity: 0 }}
+        animate={{ opacity: 1 }}
+        exit={{ opacity: 0 }}
+        transition={{
+          duration: 0.15,
+          ease: [0.4, 0, 0.2, 1],
+        }}
+      >
+        <motion.div
+          initial={{ opacity: 0, scale: 0.98, y: 5 }}
+          animate={{ opacity: 1, scale: 1, y: 0 }}
+          exit={{ opacity: 0, scale: 0.98, y: 5 }}
+          transition={{
+            duration: 0.25,
+            delay: 0.1,
+            ease: [0.4, 0, 0.2, 1],
+          }}
+          className="flex flex-col items-center gap-6"
+        >
+          <motion.div
+            initial={{ opacity: 0 }}
+            animate={{ opacity: 1 }}
+            transition={{
+              duration: 0.2,
+              delay: 0.2,
+            }}
+            className="relative"
+          >
+            <ActivityIllustration />
+          </motion.div>
+
+          <motion.div
+            initial={{ opacity: 0, y: 5 }}
+            animate={{ opacity: 1, y: 0 }}
+            transition={{
+              duration: 0.2,
+              delay: 0.25,
+            }}
+            className="flex flex-col items-center gap-1 text-center"
+          >
+            <h2 className="text-foreground-900 text-lg font-medium">
+              {emptySearchResults ? 'No activity match that filter' : 'No activity in the past 30 days'}
```

Comment:
If not mistaken, the past 30 days should either be past 30 or past 90, depending on the plan. Or we can just rephrase it to `No activity in the selected interval`.